You will need to look into the `fe/fe_values_view_{26,27,28,29}` tests and `fe/fe_values_view_number_01`. The errors appear to be of similar kind as you addressed in the library,
```
/jenkins/workspace/dealii-serial_PR-19503/tests/fe/fe_values_view_26.cc:76:63: error: ‘std::conditional<false, dealii::Tensor<1, 3>, double>::type’ {aka ‘double’} is not a class, struct, or union type
   76 |   std::vector<typename dealii::internal::CurlType<dim>::type> curls(
      |                                                               ^~~~~
```

--- a/source/fe/fe_values_views_internal.inst.in
+++ b/source/fe/fe_values_views_internal.inst.in
@@ -85,8 +85,7 @@ for (S : ALL_SCALAR_TYPES; deal_II_dimension : DIMENSIONS;
             &,
           std::vector<ProductType<
             S,
-            dealii::internal::CurlType<deal_II_space_dimension>>::type>
-            &);
+            dealii::internal::CurlType<deal_II_space_dimension>>::type> &);
 
         template void
         do_function_laplacians<deal_II_dimension, deal_II_space_dimension, S>(

--- a/tests/fe/fe_values_view_26.cc
+++ b/tests/fe/fe_values_view_26.cc
@@ -73,10 +73,9 @@ test(const Triangulation<dim> &tr, const FiniteElement<dim> &fe)
 
   // let the FEValues object compute the
   // divergences at quadrature points
-  std::vector<typename dealii::internal::CurlType<dim>::type> curls(
-    quadrature.size());
-  std::vector<Tensor<2, dim>>      grads(quadrature.size());
-  const FEValuesExtractors::Vector extractor(0);
+  std::vector<dealii::internal::CurlType<dim>> curls(quadrature.size());
+  std::vector<Tensor<2, dim>>                  grads(quadrature.size());
+  const FEValuesExtractors::Vector             extractor(0);
   fe_values[extractor].get_function_curls(fe_function, curls);
   fe_values[extractor].get_function_gradients(fe_function, grads);
 

--- a/tests/fe/fe_values_view_27.cc
+++ b/tests/fe/fe_values_view_27.cc
@@ -73,10 +73,9 @@ test(const Triangulation<dim> &tr, const FiniteElement<dim> &fe)
 
   // let the FEValues object compute the
   // divergences at quadrature points
-  std::vector<typename dealii::internal::CurlType<dim>::type> curls(
-    quadrature.size());
-  std::vector<Tensor<2, dim>>      grads(quadrature.size());
-  const FEValuesExtractors::Vector extractor(0);
+  std::vector<dealii::internal::CurlType<dim>> curls(quadrature.size());
+  std::vector<Tensor<2, dim>>                  grads(quadrature.size());
+  const FEValuesExtractors::Vector             extractor(0);
   fe_values[extractor].get_function_curls(fe_function, curls);
   fe_values[extractor].get_function_gradients(fe_function, grads);
 

--- a/tests/fe/fe_values_view_28.cc
+++ b/tests/fe/fe_values_view_28.cc
@@ -95,10 +95,9 @@ test(const Triangulation<dim> &tr, const FiniteElement<dim> &fe)
 
   // let the FEValues object compute the
   // divergences at quadrature points
-  std::vector<typename dealii::internal::CurlType<dim>::type> curls(
-    quadrature.size());
-  std::vector<Tensor<2, dim>>      grads(quadrature.size());
-  const FEValuesExtractors::Vector extractor(0);
+  std::vector<dealii::internal::CurlType<dim>> curls(quadrature.size());
+  std::vector<Tensor<2, dim>>                  grads(quadrature.size());
+  const FEValuesExtractors::Vector             extractor(0);
   fe_values[extractor].get_function_curls(fe_function, curls);
   fe_values[extractor].get_function_gradients(fe_function, grads);
 

--- a/tests/fe/fe_values_view_29.cc
+++ b/tests/fe/fe_values_view_29.cc
@@ -97,10 +97,9 @@ test(const Triangulation<dim> &tr,
 
   // let the FEValues object compute the
   // divergences at quadrature points
-  std::vector<typename dealii::internal::CurlType<dim>::type> curls(
-    quadrature.size());
-  std::vector<Tensor<2, dim>>      grads(quadrature.size());
-  const FEValuesExtractors::Vector extractor(0);
+  std::vector<dealii::internal::CurlType<dim>> curls(quadrature.size());
+  std::vector<Tensor<2, dim>>                  grads(quadrature.size());
+  const FEValuesExtractors::Vector             extractor(0);
   fe_values[extractor].get_function_curls(fe_function, curls);
   fe_values[extractor].get_function_gradients(fe_function, grads);
 

--- a/tests/fe/fe_values_view_number_01.cc
+++ b/tests/fe/fe_values_view_number_01.cc
@@ -24,12 +24,11 @@ template <typename Number>
 void
 test()
 {
-  if (typeid(typename internal::CurlType<1, Number>::type) != typeid(Number))
+  if (typeid(internal::CurlType<1, Number>) != typeid(Number))
     deallog << "NOT OK!" << std::endl;
-  if (typeid(typename internal::CurlType<2, Number>::type) != typeid(Number))
+  if (typeid(internal::CurlType<2, Number>) != typeid(Number))
     deallog << "NOT OK!" << std::endl;
-  if (typeid(typename internal::CurlType<3, Number>::type) !=
-      typeid(Tensor<1, 3, Number>))
+  if (typeid(internal::CurlType<3, Number>) != typeid(Tensor<1, 3, Number>))
     deallog << "NOT OK!" << std::endl;
   deallog << "OK" << std::endl;
 }

